lib/Services.ts

```diff
@@ -0,0 +1,12 @@
+import { Config } from "effect";
+import * as ConvertKit from "./Services/ConvertKit";
+
+export const ConvertKitLive = ConvertKit.layer({
```

Comment:
Moving use of Config to a single file allows you to quickly see all the environment vars needed in a single place.

Reply:
If there were other `Config` for other services, would it be best to define and merge all of them in a single file? Is it a best practice to keep `ConfigError` inside `Layer` instead of each `Effect`?

Reply:
I prefer to lift config errors into the layer, as it is more of a construction concern than a method concern.

Config.secret prevents values from getting into your logs, from what I recall.